docs/filesystem_layer/stratum1.md

```diff
@@ -50,8 +50,16 @@ Then install Ansible roles for EESSI:
 ansible-galaxy role install -r requirements.yml -p ./roles --force
 ```
 
+Make sure you have enough space in `/srv` (on the Stratum 1) since the snapshot of the Stratum 0
+will end up there by default. To alter the directory where the snapshot gets copied to you can add
+this variable in `inventory/host_vars/stratum1host`:
```

Comment:
The name of the file should be the same as the host/ip that you use in your hosts file (otherwise Ansible won't pick it up), so maybe it's better to use `<url-or-ip-to-your-stratum1>` here too:
```suggestion
this variable in `inventory/host_vars/<url-or-ip-to-your-stratum1>`:
```